src/mono/msbuild/android/build/AndroidApp.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <PropertyGroup>
+    <RuntimeIdentifier>$(TargetOS.ToLowerInvariant())-$(TargetArchitecture.ToLowerInvariant())</RuntimeIdentifier>
+    <UseMonoRuntime>true</UseMonoRuntime>
```

Comment:
Doesn't this get set in the highest Directory.Build.props? https://github.com/dotnet/runtime/blob/bf7cdc5825f94cf67d69ab5f0ea0c2dc9490861b/Directory.Build.props#L263

Reply:
Yes, but this could be used beyond the testing targets.  I'd prefer to leave it in.